### ISSUE TITLE
API OPTION config/scans?type=wmi invalid meta oses default

### ISSUE THREAD

**Issue by satkunas** [inverse-inc/packetfence#4301]
The following `meta` -> `oses` -> `default` uses a value ["Windows"] that is not contained in the list of allowed_lookup search_path.

```json
{
    "meta": {
        "oses": {
            "default": ["Windows"],
            "item": {
                "allowed": null,
                "allowed_lookup": {
                    "field_name": "name",
                    "search_path": "\/api\/v1\/fingerbank\/all\/devices\/search",
                    "value_name": "id"
                },
                "default": ["Windows"],
                "placeholder": null,
                "required": false,
                "type": "string"
            },
            "placeholder": null,
            "required": false,
            "type": "array"
        },
        ...
    }
    "status": 200
}```

**Comment by satkunas** [comment]
#4302 may be related

**Comment by jrouzierinverse** [comment]
fixed by 92e7bda40ec069921892e43601a006d4487f695a
